base/utils/R/utils.R

```diff
@@ -310,7 +310,7 @@ pdf.stats <- function(distn, A, B) {
     f = ifelse(B > 2,
                B/(B - 2),
                mean(stats::rf(10000, A, B))))
-  var <- switch(distn,
+  nc_var <- switch(distn,
```

Comment:
This one shouldn't be renamed -- here `var` means "variance" not "netCDF variable". But this is a great illustration of why to switch the others to unambiguous names!

Reply:
But this change uncovered a different problem! The call to the `var` function below this on line 322 ought to be spelled with its namespace included, i.e. `var(stats::rf(1e+05, A, B))))` should be changed to `stats::var(stats::rf(1e+05, A, B))))`. The variable naming was keeping `R CMD check` from noticing that before.

Reply:
> This one shouldn't be renamed -- here `var` means "variance" not "netCDF variable". But this is a great illustration of why to switch the others to unambiguous names!

I wish I would have pointed this out.